### PR TITLE
Add failure catching mechanism

### PR DESCRIPTION
**Pull request by marooned20**
Scope:
This PR adds a try-catch block to capture specific failure so that it is easier to debug in the future.

- [x] added unit tests
- [x] bumped version

### CODE REVIEW

**Review by thaisbendixen — APPROVED** [review]
LGTM!

File: tests/test_gibs.py [hunk 1]
```diff
@@ -282,6 +286,25 @@ def test_requests_wmts_tile(requests_mock):
...
 
 
+@patch("requests.get")
+@pytest.mark.parametrize(
+    "expected_error",
+    [requests.exceptions.ConnectionError(), requests.exceptions.HTTPError()],
+)
+def test_requests_wmts_tile_raises(get_mock, expected_error):
+    """
+    Mocked test for raising connection
```
Comment (markusuwe) [inline tests/test_gibs.py:296]: "...for raising exception"?

**Review by markusuwe — APPROVED** [review]
LGTM

**Review by nekhtiari — APPROVED** [review]

**Review by rodrigoalmeida94 — APPROVED** [review]
LGTM!

File: tests/test_gibs.py (hunk 1 above)
Comment (marooned20) [reply tests/test_gibs.py]: oops! Fixed it

**Pull request merged**